### PR TITLE
Uninstall helm release that was installed by kubeone but is not listed anymore in the Manifest

### LINKED ISSUE #2516: Uninstall the helm release by removing it from manifest

### PR DESCRIPTION
**Pull request by kron4eg**
**Which issue(s) this PR fixes**:
Fixes #2516 

**What type of PR is this?**
/kind feature

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
Uninstall helm release that was installed by kubeone but is not listed anymore in the Manifest
```

**Documentation**:

```documentation
https://github.com/kubermatic/docs/pull/1275
```

### CODE REVIEW

**Review by xmudrii — CHANGES_REQUESTED** [review]
Some nits, but other than that, looks good to me.

File: pkg/localhelm/helm3.go [hunk 1]
```diff
@@ -142,49 +178,127 @@ func Deploy(st *state.State) error {
...
+			return fail.Runtime(err, "initializing helm action configuration")
+		}
+
+		helmUninstall := helmaction.NewUninstall(cfg)
+		resp, err := helmUninstall.Run(rel.Name)
+		if err != nil {
+			return fail.Runtime(err, "uninstalling helm release %s/%s", rel.Namespace, rel.Name)
+		}
+
+		st.Logger.Debugf("uninstalling helm release %s/%s: %s", rel.Namespace, rel.Name, resp.Info)
```
[the remark this answers is not in the document]
Comment (kron4eg) [reply pkg/localhelm/helm3.go]: ✓

File: pkg/localhelm/helm3.go [hunk 2]
```diff
@@ -142,49 +178,127 @@ func Deploy(st *state.State) error {
...
+
+	if err := dynclient.Get(ctx, releaseNamespacedName, &releaseSecret); err != nil {
+		return fail.Runtime(err, "getting secret object for the %s secret", releaseNamespacedName)
+	}
+
+	releaseSecretOld := releaseSecret.DeepCopy()
+	if releaseSecret.Labels == nil {
+		releaseSecret.Labels = map[string]string{}
+	}
+	releaseSecret.Labels[releasedByKubeone] = "yes"
```
[the remark this answers is not in the document]
Comment (kron4eg) [reply pkg/localhelm/helm3.go]: The value here is not important absolutely, so I've replaced it with empty value.

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**